system/ee/ExpressionEngine/Addons/url/ft.url.php

```diff
@@ -56,17 +56,15 @@ public function validate($data)
         // is the scheme valid?
         if (! isset($parsed_url['host']) or ! isset($parsed_url['scheme'])) {
             // check for protocol relativity allowance before bailing
-            if (
-                in_array('//', $this->get_setting('allowed_url_schemes'))
+            if (in_array('//', $this->get_setting('allowed_url_schemes'))
```

Comment:
@dougblackjr looks like this is breaking PSR rule:
> The first expression of a multi-line control structure must be on the line after the opening parenthesis

Can you please fix this as well as resolve the conflict with changelog file